src/lib/accessibility/Behaviors/Button/ButtonGroupBehavior.ts

```diff
@@ -5,7 +5,7 @@ import { Accessibility, FocusZoneMode } from '../../interfaces'
  * Adds role 'presentation' to 'root' component's part.
  * Wraps component in FocusZone allowing arrow key navigation through the children of the component.
  */
-const ButtonGroupBehavior: Accessibility = {
+const ButtonGroupBehavior: Accessibility = (props: any) => ({
```

Comment:
Behaviors are functions, not objects. Why their names start with capital letters?

Reply:
From point of view that functions should be lowercase - yes agree with that.
I left as is just from that point how it looks
```
static defaultProps = {
    accessibility: MenuBehavior as Accessibility,
  }
```

vs

```
static defaultProps = {
    accessibility: menuBehavior as Accessibility,
  }
```

Marija has same point on that, so then I will change to lowercase

Reply:
Speaking of `as`, according to @smykhailov it is no longer necessary after @kuzhelov fixed generated paths in .d.ts.
Another thing is that `defaultProps` should be typed to components' prop interface :-)

Reply:
```Another thing is that defaultProps should be typed to components' prop interface :-)``` yeah, that I know
Ok, just to summarize, I'll make behaviors lower case.
I think defaultProps typing should be fixed as separate PR